app/models/client.rb

```diff
@@ -67,6 +67,21 @@ def self.sortable_intake_attributes
   scope :assigned_to, ->(user) { joins(:tax_returns).where({ tax_returns: { assigned_user_id: user } }).distinct }
   scope :with_eager_loaded_associations, -> { includes(:vita_partner, :intake, :tax_returns, tax_returns: [:assigned_user]) }
   scope :sla_tracked, -> { distinct.joins(:tax_returns).where.not(tax_returns: { status: TaxReturnStatus::EXCLUDED_STATUSES })}
+  scope :outgoing_communication_breaches, ->(breach_threshold_datetime) do
+    sla_tracked.where(arel_table[:first_unanswered_incoming_interaction_at].lteq(breach_threshold_datetime))
```

Comment:
Consider also `.where("first_unanswered_incoming_interaction_at <= ?", breach_threshold_datetime)` but not a blocker.